### ISSUE TITLE
Cancel button only changes the url and scrolls up when using tabs with persistTabInQueryString

### ISSUE THREAD

**Issue by lotarbo** [laravel-filament/filament#14332]
### Package

filament/filament

### Package Version

v3.2.114 

### Laravel Version

v11.24.0

### PHP Version

PHP 8.2

### Problem description

The Cancel button only scrolls to the top and changes the url. And only after a few clicks it returns to the list page
I use Tabs with persistTabInQueryString()

https://github.com/user-attachments/assets/269cb4e1-5ddb-4b60-9b36-c5094a4b5ee4

I think related to https://github.com/filamentphp/filament/issues/11460

### Expected behavior

I think the first time you enter an entity - cancel should return back to the list.
And if we go through tabs - either return to the list or switch tabs in reverse order. 
Although it seems logical to return to the list at once, it's not a back button as it seems to me.

### Steps to reproduce

1. Open Dashboard
2. Go to "Pages"
3. Click "New page" button
4. Click To tab "SEO"
5. Click "Cancel" button
6. Click "Cancel" button two times
7. We back on List page

### Reproduction repository (issue will be closed if this is not valid)

https://github.com/lotarbo/filament-issue-cancel

**Comment by zepfietje** [comment]
Duplicate of https://github.com/filamentphp/filament/issues/14085.
